config/dev/aws-managedcluster.yaml

```diff
@@ -13,7 +13,7 @@ spec:
       instanceType: t3.small
     controlPlaneNumber: 1
     publicIP: true
-    region: us-west-2
+    region: ${AWS_REGION}
```

Comment:
I think EKS YAMLs also would benefit from the environ

Reply:
Done.